src/main/java/com/google/devtools/build/lib/bazel/bzlmod/AttributeValuesAdapter.java

```diff
@@ -88,9 +90,11 @@ private JsonElement serializeObject(Object obj) {
         jsonObject.add(serializeObjToString(entry.getKey()), serializeObject(entry.getValue()));
       }
       return jsonObject;
-    } else if (obj instanceof StarlarkList) {
+    } else if (obj instanceof StarlarkList || obj instanceof Tuple) {
```

Comment:
looking at [the implementation of ListType](https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/packages/Type.java;drc=7c89c7f4de678efb76849b68647d57900779c48e;l=674), all Iterables should be accepted. (Technically also Depset, but `depset` is not available in MODULE.bazel so I'm not worried.) Could we switch this to Iterable instead?

Reply:
How about `Sequence`? That should cover everything expected exposed to Starlark. Everything else should probably rightfully result in a crash.

Reply:
Nvm, you are right, let's go with Iterable. Makes it easier to write the comment as well :-)